documentation/README.md

```diff
@@ -871,13 +872,10 @@ and returns `undefined` if the `predicate` is false.
 ```js
 const ifThen = require('1-liners/ifThen');
 
-const eq = (a, b) => a === b;
-const add = (a, b) => a + b;
-const sub = (a, b) => a - b;
```

Comment:
Don't why it is getting deleted while i commit. But this piece code is not no the master branch.